src/PassportServiceProvider.php

```diff
@@ -227,6 +231,20 @@ protected function registerClientRepository()
         });
     }
 
+    /**
+     * Register the JWT Parser.
+     *
+     * @return void
+     */
+    protected function registerJWTParser()
+    {
+        if (class_exists(JWTTokenParser::class)) {
+            $this->app->singleton(JWTParser::class, function () {
+                return new JWTTokenParser(new JoseEncoder());
+            });
+        }
```

Comment:
Using the configuration object here would simplify things:

```suggestion
        $this->app->singleton(Parser::class, function () {
            return Configuration::forUnsecuredSigner()->parser();
        });
```